Libraries/Vibration/VibrationIOS.android.js

```diff
@@ -6,18 +6,28 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
- * Stub of VibrationIOS for Android.
- *
- * @providesModule VibrationIOS
+ * @providesModule VibrationAndroid
  */
+
 'use strict';
 
-var warning = require('warning');
+var RCTVibrationAndroid = require('NativeModules').VibrationAndroid;
```

Comment:
Please don't use the `RCT` prefix in JS, it's an Obj-C convention we're still using in JS and we should move away from it.